app/src/main/java/com/hover/stax/pushNotification/FirebaseMessageReceiver.java

```diff
@@ -0,0 +1,74 @@
+package com.hover.stax.pushNotification;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.widget.RemoteViews;
+
+import androidx.annotation.NonNull;
+import androidx.core.app.NotificationCompat;
+
+import com.google.firebase.messaging.FirebaseMessaging;
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+import com.hover.stax.R;
+import com.hover.stax.SplashScreenActivity;
+import com.hover.stax.utils.Constants;
+import com.hover.stax.utils.DateUtils;
+
+public class FirebaseMessageReceiver extends FirebaseMessagingService {
+	@Override
+	public void onNewToken(@NonNull String s) {
+		super.onNewToken(s);
+	}
+
+	@Override
+	public void
+	onMessageReceived(RemoteMessage remoteMessage) {
+		if (remoteMessage.getNotification() != null) {
+			showNotification(remoteMessage.getNotification().getTitle(), remoteMessage.getNotification().getBody());
+		}
+	}
+
+	private RemoteViews getCustomDesign(String title, String message) {
```

Comment:
Why are we making our own design rather than using the default which already contains a title, message and icon?

Reply:
No data to support this: but personally, I tend to engage with notifications that have a custom layout, cos they are just different from regular ones; eyes tend to focus on the different ones.

I think this might help our engagement, but I'd put this in Pivotal for Justin.